progress/style.go

```diff
@@ -157,6 +160,7 @@ var (
 	// example to customize the Tracker rendering.
 	StyleOptionsDefault = StyleOptions{
 		DoneString:              "done!",
+		ErrorString:             "fail!",
```

Comment:
`fail` to `failed`?

Reply:
fail is the same number of characters as done, so the output lines up nicely :)

Reply:
Gotcha! Let's leave it like that.